stubs/mock/mock/mock.pyi

```diff
@@ -378,3 +379,13 @@ class PropertyMock(Mock):
     def __set__(self, obj: Any, value: Any) -> None: ...
 
 def seal(mock: Any) -> None: ...
+
+class ThreadingMixin(Base):
+    DEFAULT_TIMEOUT: float | None
```

Comment:
The docstring says:

> You can change the default timeout of all instances via the `ThreadingMock.DEFAULT_TIMEOUT` attribute.

https://github.com/testing-cabal/mock/blob/d344fa2794b3b1ae7e4a4dbf265fb040d6f41d1f/mock/mock.py#L3137C1-L3138C47

```suggestion
    DEFAULT_TIMEOUT: ClassVar[float | None]
```

Reply:
Yeah, this is a preferable way: how it should be used.
Right now it is annotated the way it can be used: https://github.com/testing-cabal/mock/blob/d344fa2794b3b1ae7e4a4dbf265fb040d6f41d1f/mock/mock.py#L3071